File: DailyFlash Solutions/DailyFlash Solutions/64-DailyFlash_Solutions/03_Apr_Solution_One/Python/Program0.py
from math import *

n = int(input("Enter the limit : "))
x,y = [ int(x) for x in input("Enter x and y : ").split()]

s=0.0

for i in range(1,n+1):
        s+=1/(((x+y)**i)*factorial(i))


print("Result of series is :",round(s,2))

File: DailyFlash Solutions/DailyFlash Solutions/54-DailyFlash_Solutions/23_March_Solution_Four/C/Program0.c
#include <stdio.h>
#include <math.h>

void main() {

	double l = 0,sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%lf",&l);
	

	for(double i = 1; i <= l; i++)
		sum+=i/sqrt(i*(i+1));
	
	printf("Sum of Series = %.2lf\n",sum);
}
